systemd/systemd.go

```diff
@@ -94,6 +85,15 @@ type Collector struct {
 
 // NewCollector returns a new Collector exposing systemd statistics.
 func NewCollector(logger log.Logger) (*Collector, error) {
+	// We could add a cpu label, but IMO that could cause a cardinality explosion. We already export
```

Comment:
It may be nice to add the user's username to this comment. Moving it means it will be harder to track the history of who made this comment in git.

Reply:
Yea, I guess I could move this section back, but the OCD in me wanted these in alphabetical order. :)